src/core/modules/MinorPlanet.hpp

```diff
@@ -93,6 +93,9 @@ class MinorPlanet : public Planet
 	//! @param slope Slope parameter G. This is usually [0..1], sometimes slightly outside. Allowed here [-1..2].
 	void setAbsoluteMagnitudeAndSlope(const float magnitude, const float slope);
 
+	//! sets radius based on absolute magnitude.
```

Comment:
... based on absolute magnitude and albedo (assume default 0.15)

Reply:
Updated comment. Uses ssystem_minor.ini albedo if provided.